pkgs/applications/window-managers/dwl/default.nix

```
@@ -8,9 +8,9 @@
 , pkg-config
 , wayland
 , wayland-protocols
-, wlroots
+, wlroots_0_16
 , writeText
-, enable-xwayland ? true, xwayland, libX11
+, enable-xwayland ? true, xorg, libX11
```

```suggestion
, enableXWayland ? true, xorg, libX11
```

enable-… is not nixpkgs style, and wlroots already spells it `enableXWayland` too.